app/views/layouts/application.html.erb

```diff
@@ -1,41 +1,48 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>RorSocialScaffold</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
-  <body>
-    <nav class="nav">
-      <%= link_to 'Stay in touch', root_path %>
-      <div class="menu">
-        <div class="menu-item">
-          <%= menu_link_to 'Timeline', posts_path %>
-        </div>
-        <div class="menu-item">
-          <%= menu_link_to 'All users', users_path %>
-        </div>
+
+<head>
+  <title>RorSocialScaffold</title>
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  <%= stylesheet_link_tag 'https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css' %>
+</head>
+
+<body>
+  <nav class="nav">
+    <%= link_to 'Stay in touch', root_path %>
+    <div class="menu">
+      <div class="menu-item">
+        <%= menu_link_to 'Timeline', posts_path %>
       </div>
-      <% if current_user %>
-        <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
-      <% else %>
-        <%= link_to 'Sign in', user_session_path %>
-      <% end %>
-    </nav>
-    <% if notice.present? %>
-      <div class="notice">
-        <p> <%= notice %> </p>
+      <div class='menu-item'>
+        <%= menu_link_to 'All users', users_path %>
       </div>
+    </div>
+    <% if current_user %>
+    <%= link_to current_user.name, user_path(current_user) %> |
+    <%#= gravatar_image_tag(current_user.email, :gravatar => { :size => 20 }, alt: current_user.name) %>
+    <%#= link_to gravatar_image_tag(current_user.email, gravatar: { :size => 15 }, alt: current_user.name) %>
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
See this [post](https://stackoverflow.com/questions/13130859/inline-comments-in-ruby/13130900#:~:text=No%2C%20Ruby%20does%20not%20have,comment%20out%20the%20one%20row), about why comments are discouraged.